src/main/java/tictactoe/Board.java

```diff
@@ -97,9 +96,14 @@ private int getSide() {
         return (int)Math.sqrt(grid.size());
     }
 
-    public void markCellAtPosition(String mark, int position) {
-        Cell cell = grid.get(position - 1);
-        cell.setMark(mark);
+    public void markCellAtPosition(String mark, String input) {
```

Comment:
With this change, your Board is now coupled to how a user enters a move.

If we wanted to change how to communicate the next (let's say I want to use "top-left", "top-centre", etc.), the Board class would be affected.
It would be better if the Board was not aware of Input/Output details.

Also notice that you have added a responsibility to this Class; this added responsibility could be described as "the Board knows how to parse String user input into a move".